Sources/SWBUtil/Path.swift

```diff
@@ -69,18 +69,45 @@ public struct Path: Serializable, Sendable {
     /// The system path separator.
     #if os(Windows)
     public static let pathSeparator = Character("\\")
-    public static let pathSeparatorUTF8 = UInt8(ascii: "\\")
-    public static let pathSeparatorsUTF8 = Set([UInt8(ascii: "\\"), UInt8(ascii: "/")])
+    @inline(__always) public static var pathSeparatorUTF8: UInt8 { UInt8(ascii: "\\") }
     public static let pathEnvironmentSeparator = Character(";")
-    public static let pathSeparators = Set("\\/")
+    @inline(__always) public static func isUTF8PathSeparator(_ char: UInt8) -> Bool {
+        guard let separators else {
+            return char == pathSeparatorUTF8 || char == UInt8(ascii: "/")
+        }
+        // This is a bit inefficient, but separators should always be nil outside of tests
+        return separators.contains(String(decoding: CollectionOfOne(char), as: UTF8.self))
+    }
+    public static func firstPathSeparatorIndex(in str: some StringProtocol, separators: (some Collection<Character>)?) -> String.Index? {
```

Comment:
I pulled this out into its own function so that I could use firstIndex(where:) (which has to deal with a closure) on Windows, and index(of:) everywhere else, since only Windows has more than one value to check for